common/djangoapps/util/tests/test_db.py

```diff
@@ -120,6 +121,7 @@ class MigrationTests(TestCase):
     Tests for migrations.
     """
 
+    @skip
```

Comment:
We should have this changed back when you introduce the migration file with the next PR, right?

Reply:
Yes, we will changed back this with the next PR.